src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
> Is this client version language-dependent?

Yes. For Java, this is configured in the Packman configuration. For Python/NodeJS/Ruby, this is configured in googleapis/gapic/packaging.

> Ideally we should have the generator and the generated client in sync, or at least having some coordination. This means if a user uses a X version generator, he should be able to generate the X version client.

What if the API introduces a new method, but the generator doesn't change? We should be able to tell that the user is using a client library with the new method, even it's the same generator as previously.